pypif/obj/common/property.py

```diff
@@ -79,6 +82,19 @@ def method(self):
         self._method = None
 
     @property
```

Comment:
I can't comment on the lines that I want, so I will here. Taking `def method(self)` as an example, we should "deprecate" those. For example:
```
def method(self):
    from warnings import warn
    warn("method has been deprecated in favor of methods")
    return self._methods
```
The import statement can go to top of file.